src/Altinn.App.Core/Configuration/AppSettings.cs

```diff
@@ -61,8 +62,13 @@ public class AppSettings
         /// <summary>
         /// Gets or sets the BaseResourceFolderContainer that identifies where in the docker container the runtime can find files needed
         /// </summary>
+        // TODO: can this be removed?
+        // Env var being set is ServiceRepositorySettings__BaseResourceFolderContainer, but this prop is not used anywhere
+#nullable disable
         public string BaseResourceFolderContainer { get; set; }
 
+#nullable restore
```

Comment:
Yeah it's temporary, idea being that we already use `#nullable disable` a lot of places, so that for the next major bump we will just search through all of these. I also want to minimize null-forgiving operator as I kind of want to use `Nullable.Extended.Analyzer` too